### ISSUE TITLE
Is there a plan to add Wan2.1 VACE support?

### ISSUE THREAD

**Issue by ShaoDingBao** [a-r-r-o-w/cogvideox-factory#403]

**Comment by a-r-r-o-w** [comment]
I am working on the integration for diffusers at the moment. Will follow up with training soon after

**Comment by ShaoDingBao** [comment]
Thank you for your answer.

**Comment by Auxotaku** [comment]
Thanks for your work. I have seen the VACE model you uploaded on HF, but the input of VACE requires a combination of src_video src_mask src_ref_image. Will this cause difficulties for fine-tuning? I haven't seen its fine-tuning code yet.

> I am working on the integration for diffusers at the moment. Will follow up with training soon after

**Comment by ShaoDingBao** [comment]
@a-r-r-o-w 
Hello,
Are there any recent updates or developments regarding the VACE training code?
Thanks!

**Comment by a-r-r-o-w** [comment]
Hi, I've tried applying just video based training (ignoring the reference images) on the denoiser part of the model. It trains successfully but the results are not great unfortunately

**Comment by ShaoDingBao** [comment]
> Hi, I've tried applying just video based training (ignoring the reference images) on the denoiser part of the model. It trains successfully but the results are not great unfortunately

Hi, could you please clarify what you mean by "video-based training"? On which specific task did you train and evaluate the model? As far as I understand, VACE is a controllable video generation framework, and I'm looking to adapt it to my own specific conditional generation task. I have prepared reference images, the original video, and corresponding masks. Does the current VACE training framework support incorporating such diverse control conditions to train a task-specific VACE model?

**Comment by ShaoDingBao** [comment]
Excuse me for bothering you again. Could you please provide an update on the current progress of the VACE training code? Do you have a definite timeline for it? @a-r-r-o-w
